src/main/resources/jenkins/branch/EmptyOrganizationFolderView/main.jelly

```diff
@@ -0,0 +1,44 @@
+<!--
+ ~ The MIT License
+ ~
+ ~ Copyright (c) 2015, CloudBees, Inc.
+ ~
+ ~ Permission is hereby granted, free of charge, to any person obtaining a copy
+ ~ of this software and associated documentation files (the "Software"), to deal
+ ~ in the Software without restriction, including without limitation the rights
+ ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ ~ copies of the Software, and to permit persons to whom the Software is
+ ~ furnished to do so, subject to the following conditions:
+ ~
+ ~ The above copyright notice and this permission notice shall be included in
+ ~ all copies or substantial portions of the Software.
+ ~
+ ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ ~ THE SOFTWARE.
+ -->
+<?jelly escape-by-default='true'?>
+<st:compress xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+  <div style="margin: 20px 0; padding: 10px 30px 20px 30px; background: #f8f8f8; border-radius: 10px; max-width: 55em;">
+    <h1>This folder is empty</h1>
+    
+    <p>
+        There are no repositories found that contain buildable projects.
+        Organization Folders automatically build and manage repositories that
+        contain recognizable projects.
+    </p>
+    
+    <j:forEach var="projectFactory" items="${it.owner.projectFactories}">
+        <st:include it="${projectFactory}" page="getting-started" optional="true"/>
```

Comment:
:ant: `MultiBranchProjectFactory` class Javadoc should mention that it may have a view named `getting-started`.